src/OrchardCore.Modules/OrchardCore.Lists/Views/ListPartDetailAdmin.cshtml

```diff
@@ -76,9 +76,9 @@ else
                             }
                             else if (authorizedContentTypeDefinitions.Count() > 1)
                             {
-                                <a class="btn btn-sm btn-success dropdown-toggle" href="#" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                                <button class="btn btn-sm btn-success dropdown-toggle" type="button" id="dropdownMenuLink" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```

Comment:
Do you really need `type="button"` here since it is a button?

Reply:
No need

Reply:
Removed `type="button"` https://github.com/OrchardCMS/OrchardCore/pull/11860/commits/615b6f39db13ab777e87b4784d969eda06593f7c